src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -51,7 +51,7 @@
 from pants.option.option_types import BoolOption, EnumOption, IntOption
```

Comment:
Where your journey with options all began 🥰

Reply:
It really was from all the name changes on my "asset inference" PR that started everything :joy: